drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -642,7 +653,10 @@ def _get_item_definition(self, item_id):
         """
         Returns definition (settings) for item identified by `item_id`.
         """
-        return next(i for i in self.data['items'] if i['id'] == item_id)
+        try:
+            return next(i for i in self.data['items'] if i['id'] == item_id)
+        except StopIteration:
+            raise Exception(self.data['items'], item_id)
```

Comment:
@e-kolpakov Under what conditions do we expect this exception to fire? It might be worth adding comment about that here.